src/app/components/Article/index.jsx

```diff
@@ -35,18 +49,18 @@ class Article extends Component {
   }
 
   render() {
-    const { headline } = this.state;
+    const { data } = this.state;
+    const { text } = data.blocks[0].model.blocks[0].model.blocks[0].model;
```

Comment:
Rather than extracting a nested `text` block, I think we should add a generic `title` property [here](https://github.com/bbc/simorgh/pull/247/files#diff-5f296fd8afe33a8853c3f2655172c257R9).